Makefile

```diff
@@ -27,5 +27,11 @@ restart:
 lint: isort black
 	@true
 
+isort:
+	isort . --check --diff; 
+
+black:
+	black . --check;
```

Comment:
```suggestion
	black . --check
```

Reply:
Why no semicolon on this one?

Reply:
i meant to remove the semicolon on the other one as well, i don't think it's necessary

Reply:
I'll check to make sure when I'm back